src/lib/pages/backoffice/DocumentRequest/DocumentRequestDetails/DocumentRequestSteps/DocumentStep/DocumentStep.js

```diff
@@ -34,17 +33,12 @@ DocumentStep.propTypes = {
 };
 
 export default class DocumentStepContent extends Component {
-  onSelectResult = async data => {
+  onSelectResult = data => {
     const {
-      fetchDocumentRequestDetails,
```

Comment:
the `fetchDocumentRequestDetails` is now unused prop (check), if yes, please remove it from `propTypes` and mapping in the `index.js`
you should get build  errors for unused props after you rebase from master